src/main/java/com/mcmoddev/basemetals/init/Fluids.java

```diff
@@ -40,17 +41,15 @@ public static void init() {
 				MaterialNames.PEWTER, MaterialNames.PLATINUM, MaterialNames.SILVER, MaterialNames.STARSTEEL,
 				MaterialNames.STEEL, MaterialNames.TIN, MaterialNames.ZINC);
 
-		vanillaMaterialsMap.forEach(name -> {
+		vanillaMaterialsMap.stream().filter(Options::isFluidEnabled).forEach(name -> {
 			addFluid(name, 2000, 10000, 769, 10);
```

Comment:
I suggest maybe use constant values for some of these numeric input parameters?

Reply:
We can fix that in a later patch, it doesn't regress anything currently, I do agree with you though, we can do it like we do with the materials.